tests/distributed_mode/test_pipeline.py

```diff
@@ -53,6 +59,8 @@ def test_subpipeline_is_not_recreated_in_remote_workers():
     assert len(subpipelines) == 1
     assert len(documents) == 1
     assert len(nlp.pipeline[0].keys()) == 2
+    assert list(nlp.pipeline[0].keys())[1] == "alice"
```

Comment:
Keys in a dictionary are not guaranteed to be ordered in the same order you added them, so this code `list(nlp.pipeline[0].keys())[1]` is risky, `alice` might not be the second in the list even if it is added after bob. The right way to test would be 

```
assert "alice" in nlp.pipeline[0]" #no need to call .keys() in any of the above code or here
```

Reply:
Would you please consider this comment

Reply:
Yes, I have made the changes to my local repo. I was waiting for your answer on my other doubt. So that I could push them together as a single commit.

Reply:
@sachin-101 Great, I will merge whenever you push the changes